Dockerfile

```
@@ -18,8 +18,49 @@
+  luarocks \
+  git \
+  # apt clean-up
+  && apt-get autoremove -y \
+  && rm -rf /var/lib/apt/lists/*
+
+ARG NEOVIM_VERSION
+
+WORKDIR /tmp
+
+# Install NEOVIM
+RUN if [ "$NEOVIM_VERSION" = "stable" ]; then \
```

Hadolint (2.12.0)

[error] Expected 'fi'. Fix any mentioned problems and try again.

(SC1072)